mu-plugins/plugin-tweaks/index.php

```diff
@@ -0,0 +1,49 @@
+<?php
```

Comment:
>  would it be better to have each tweaked plugin have its own file even if there's only one function?

I'd lean towards that. Over time the file will get big if there's only one, and become unwieldy. We could break off a few separate files for groups of tweaks at that point, but then `index.php` wouldn't be named descriptively. 

Having a separate file also makes it show up in IDE searches, which can help devs learn about it if they didn't already know (or forgot) that it exists. 

So something like:

```
mu-plugins/plugin-tweaks/jetpack.php
mu-plugins/plugin-tweaks/gutenberg.php
mu-plugins/plugin-tweaks/camptix.php
```

Not a big deal either way though.

Reply:
Good points. I think this is a better approach.